self-hosted-login/okta-aspnetcore-mvc-example/Startup.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Collections.Generic;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Okta.AspNetCore;
+using okta_aspnetcore_mvc_example.Models;
+
+namespace okta_aspnetcore_mvc_example
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.Configure<OktaSettings>(Configuration.GetSection("Okta"));
+
+            var oktaMvcOptions = new OktaMvcOptions()
+            {
+                OktaDomain = Configuration.GetSection("Okta").GetValue<string>("OktaDomain"),
+                ClientId = Configuration.GetSection("Okta").GetValue<string>("ClientId"),
+                ClientSecret = Configuration.GetSection("Okta").GetValue<string>("ClientSecret"),
+                Scope = new List<string> { "openid", "profile", "email" },
```

Comment:
Are the defaults okay here? In that case could we omit Scope and GetClaimsFromUserInfoEndpoint from config?